src/vaillant_netatmo_api/thermostat.py

```diff
@@ -93,6 +95,65 @@ async def async_get_thermostats_data(self) -> list[Device]:
 
         return [Device(**device) for device in body["body"]["devices"]]
 
+    async def async_get_dhw_data(
+        self,
+        device_id: str,
+        home_id: str,
+    ) -> None:
+        """
+        Get DHW data from the Netatmo API.
+
+        """
+
+        path = _GET_DHW_DATA_PATH
+        data = {
+            "device_id": device_id,
+            "home_id": home_id,
+        }
+
+        body = await self._post(
+            path,
+            data=data,
+        )
+
+        if body["status"] != _RESPONSE_STATUS_OK:
+            raise NonOkResponseException("Unknown response error. Check the log for more details.", path=path, data=data, body=body)
+
+        return body["body"]["home"]["modules"][0]["dhw_always_on"]
```

Comment:
Instead of returning just one data point, the API should be generic and return the whole object, properly decoded into its own class.

Suggestion here is to create three new classes: `Config`, `ConfigHome` and `ConfigModule`. The naming is bad, but I'd leave it at that for now. The reason is because, now that I see this new API, I'd like to refactor the code a bit to properly align with the fact that there are _two_ modules, one as a return value of `getconfig` API call and other as a return value of `getthermostatasdata` API endpoint.

But this refactor should be done separately not to increase the code change in this one.

Reply:
Separated classes would be great. So should I cancel this PR and wait for your changes?

Reply:
I suggest you add those classes and change the code to return them. You can use `getthermostatdata` API call as a reference, since it will be very similar to that one.

Once you are done and we merge your PR which adds the feature, I'll take some time to refactor it. Maybe not immediately, but some time in the future, as functionality-wise, this will work nicely.